app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+def create_report(data_file_name: str, report_file_name: str):
+    with open(data_file_name, "r") as data:
+        lines = data.readlines()
+
+    supply = []
+    buy = []
+
+    for line in lines:
+        if line.startswith("supply"):
+            for idx, value in enumerate(line):
+                if value == ",":
```

Comment:
It is not good to find comma here. It is `comma separated values` and if there was more values your code would not work. It would be relevant here to split line by comma.

Reply:
Fixed